system/cc-ceph/values.yaml

```
@@ -207,6 +207,16 @@ objectstore:
       disks: "*"
       nodeSelector:
         kubernetes.metal.cloud.sap/role: ceph-osd
+    rgwSidecarWebhook:
+      enabled: false
+      namespace: "rook-ceph"
+      image: "keppel.eu-de-1.cloud.sap/ccloud/prysm-wh"
+      tag: "20250221130000"
```

it's better to align with other image references, then adjust the templates accordingly.
```suggestion
      repository:
        image: "keppel.eu-de-1.cloud.sap/ccloud/prysm-wh"
        tag: "20250221130000"
```